$OpenBSD$
Add _SAMBA4 sufffix to generated routines to de-conflict with base system
Heimdal installation.

Note: we do not touch some routines like add_*, they does not end in nm(1)
output so probably they are safe.
--- source4/heimdal/lib/asn1/gen_seq.c.orig	Sat May 18 20:25:03 2013
+++ source4/heimdal/lib/asn1/gen_seq.c	Sat May 18 20:27:42 2013
@@ -85,7 +85,7 @@ generate_type_seq (const Symbol *s)
 	     "\t(data->len + 1) * sizeof(data->val[0]));\n"
 	     "if (ptr == NULL) return ENOMEM;\n"
 	     "data->val = ptr;\n\n"
-	     "ret = copy_%s(element, &data->val[data->len]);\n"
+	     "ret = copy_%s_SAMBA4(element, &data->val[data->len]);\n"
 	     "if (ret) return ret;\n"
 	     "data->len++;\n"
 	     "return 0;\n",
@@ -103,7 +103,7 @@ generate_type_seq (const Symbol *s)
 	     "\n"
 	     "if (data->len == 0 || element >= data->len)\n"
 	     "\treturn ASN1_OVERRUN;\n"
-	     "free_%s(&data->val[element]);\n"
+	     "free_%s_SAMBA4(&data->val[element]);\n"
 	     "data->len--;\n"
 	     /* don't move if its the last element */
 	     "if (element < data->len)\n"
